### DISCUSSION TITLE
Lit the icon when the cruise control is active

### DISCUSSION THREAD

**Discussion by VenoushCZ** [jason0611/FS22_DashboardLive#39] [Ideas]
Hi! Would it be possible to lit an icon when the cruise control is active? It would help very much during the no-HUD operations.

**Comment by VenoushCZ** [comment]
I think I have found the answer:

https://github.com/jason0611/FS22_Xerion_DBL/blob/master/xerion5000.xml

            <dashboard displayType="TEXT" valueType="cruiseControl" node="numbersCruiseControl01" textColor="0.1 0.1 0.1 1" textSize="0.005" textMask="00.0" textAlignment="RIGHT" textScaleX="0.88" font="GENERIC" groups="MOTOR_ACTIVE"/>
            <dashboard displayType="VISIBILITY" valueType="cruiseControlActive" node="dbl_cruise" groups="MOTOR_ACTIVE"/>
			<dashboard displayType="EMITTER" valueType="cruiseControlActive" node="dbl_cruise" baseColor="BLACK" emitColor="YELLOW" intensity="0.8" groups="MOTOR_ACTIVE" />

**Comment by jason0611** [comment]
Yes, that's a basic game feature. ;-)
